### DISCUSSION TITLE
Add STM32F030CCT6 IAP example code

### DISCUSSION THREAD

**Discussion by xiezhoubin** [stm32duino/Arduino_Core_STM32#2433] [General]
I have tried to write the STM32F030CCT6 IAP function, but have not been successful. I hope the official website can provide example code.

**Comment by fpistm** [comment]
Hi @xiezhoubin 

this kind of request is not an issue that's why I convert it as a discussion, you can also use the forum to request help form other member.
Unfortunately, I have no time to address such request.
